cli/src/main/java/com/devonfw/tools/ide/context/AbstractIdeContext.java

```diff
@@ -737,8 +742,13 @@ public void logIdeHomeAndRootStatus() {
     if (this.ideRoot != null) {
       success("IDE_ROOT is set to {}", this.ideRoot);
     }
+
     if (this.ideHome == null) {
-      warning(getMessageIdeHomeNotFound());
+      if (this.getIdeRootPathFromEnv() != null && this.getIdeRootPathFromEnv().equals(this.ideRoot)) {
+        warning(getMessageIdeHomeNotSet());
+      } else {
+        warning(getMessageIdeHomeNotFound());
+      }
```

Comment:
This change has nothing to do with bug #1060 
I do not get the purpose of this change.
I agree that the message 
```
You are not inside an IDE installation: C:\projects
```
may be misleading and could be improved to
```
You are not inside an IDE project: C:\projects
```

But for me this change esp. with this fine grained distinction of the same purpose with 2 different messages does not seem to make sense.